app/api/emails.py

```
@@ -0,0 +1,141 @@
+        
+        if not email:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail=f"Email {email_id} not found"
+            )
+        
+        # Parse to_addresses with error handling
+        try:
+            parsed_to = json.loads(email.to_addresses)
+        except (json.JSONDecodeError, TypeError) as parse_error:
+            logger.exception(f"Error parsing to_addresses for email {email_id}: {str(parse_error)}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)